lib/gtfs/reducers/filter.js

```diff
@@ -8,6 +8,7 @@ import type {FilterState} from '../../types/reducers'
 
 export const defaultState = {
   activeFeeds: {},
+  comparedVersion: null,
```

Comment:
Please rename to `comparedVersionId` to reflect that it is an ID being stored here.

Reply:
Only the version object is stored as of https://github.com/ibi-group/datatools-ui/pull/528/commits/03695eda1153b68d93751b1f0d27738edc2b41b1.